accessibilityTest/AccessibilityDynamicHierarchicalFacet.ts

```diff
@@ -0,0 +1,32 @@
+import * as axe from 'axe-core';
+import { $$, Component, DynamicHierarchicalFacet } from 'coveo-search-ui';
+import { afterQuerySuccess, getFacetColumn, getRoot, inDesktopMode, resetMode } from './Testing';
+
+export const AccessibilityDynamicHierarchicalFacet = () => {
+  describe('DynamicHierarchicalFacet', () => {
+    let dynamicHierarchicalFacet: HTMLElement;
+
+    beforeEach(() => {
+      dynamicHierarchicalFacet = $$('div', {
+        className: Component.computeCssClassName(DynamicHierarchicalFacet),
+        dataTitle: 'My Facet',
+        dataField: '@objecttype',
+        dataEnableCollapse: true
+      }).el;
+
+      inDesktopMode();
+    });
+
+    afterEach(() => {
+      resetMode();
+    });
+
+    it('should be accessible', async done => {
```

Comment:
Could also add some test with some interaction ? Since it changes the DOM a bit, it would be good to have minimal testing for example when you select a value, if you minimize and expand, etc.

Reply:
There's an optimal test that is when a user clicks on a result. There is the additional All Categories button that is added. Other than that, the DOM doesn't change much, the values are hierarchical but the DOM is a list.